static/script.js

```diff
@@ -18,20 +17,43 @@
   const AGENCY_KEY = "AGBUR";
   const MIN_ZOOM = 6;
 
-  L.tileLayer(
+  // create the tile layer with correct attribution
+
+  var hotLayer = new L.tileLayer('http://{s}.tile.openstreetmap.fr/hot/{z}/{x}/{y}.png', 
+  	{
+  		maxZoom: 20, 
+  		attribution: 'Data \u00a9 <a href="http://www.openstreetmap.org/copyright"> OpenStreetMap Contributors </a> Tiles \u00a9 HOT'
+  	}),
+  darkmatterLayer = L.tileLayer(
     "https://cartodb-basemaps-{s}.global.ssl.fastly.net/dark_nolabels/{z}/{x}/{y}.png",
     {
       maxZoom: 18,
       attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>, &copy; <a href="https://carto.com/attribution">CARTO</a>'
-    }
-  ).addTo(myMap);
-
-  const labelLayer = L.tileLayer(
-    "https://cartodb-basemaps-{s}.global.ssl.fastly.net/dark_only_labels/{z}/{x}/{y}.png",
+    });
+  var thunderforest = L.tileLayer(
+    "https://a.tile.thunderforest.com/landscape/{z}/{x}/{y}.png",
     {
-      maxZoom: 18
-    }
-  ).addTo(myMap);
+      maxZoom: 18,
+      attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>, &copy; <a href="https://carto.com/attribution">CARTO</a>'
+    });
+  
+  var myMap = L.map("map",
+  {
+  	center: [38, -98],
+  	zoom: 4,
+  	layers: [hotLayer, darkmatterLayer, thunderforest] //multiple layers by default
+  });
+  L.hash(myMap); // enable hashed location
+
+  var baseLayers = {
+  	"hotLayer": hotLayer,
+  	"darkmatterLayer": darkmatterLayer,
+    "thunderforest": thunderforest,
+  };
+  var controlLayers = L.control.layers(baseLayers).addTo(myMap);
```

Comment:
I think some styling work needs to be done for the layer control. It currently looks like:
<img width="235" alt="screen shot 2017-05-26 at 12 49 54 pm" src="https://cloud.githubusercontent.com/assets/697848/26506369/e96faaa6-4211-11e7-80b3-ef2b7604d4ad.png">

But it should look like a list of radio buttons. I am pretty sure the problem is that the Web Design Standards css is bonking this control's styling, so it probably just needs a little work in `static/style.css` to get it looking correct.

Should look something like:
<img width="134" alt="screen shot 2017-05-26 at 12 51 59 pm" src="https://cloud.githubusercontent.com/assets/697848/26506429/26656086-4212-11e7-94b6-4774af1af0db.png">